### ISSUE TITLE
image build skipped should not be pushed by default

### ISSUE THREAD

**Issue by ebaotech-oss** [fabric8io/docker-maven-plugin#1243]
Hi,

Currently, when one of the image build is skipped (example below), push will still try to push all two images, is it possible for push to honor image.build.skip property?

<images>
    <image>
      <build>
        <name>image1</name>
        <skip>${maybe.skipped.1}</skip>
     </build>
   </image>
    <image>
      <build>
        <name>image2</name>
        <skip>${maybe.skipped.2}</skip>
     </build>
   </image>
<images>

Thanks

**Comment by rhuss** [comment]
I agree that there should be a possibility to skip the push for certain images. However, I would introduce a `skipPush` field (like `skipTag`), to deliberately specify that intent. This is mostly to not break backwards compatibility.

Unfortunately I don't have any spare time at the moment, so if you are interested in that feature, feel free to submit a pull request and I will help you to integrate it.
